doc/source/downloading_cesm.rst

```diff
@@ -120,50 +107,51 @@ columns of output, as in this example:
        ./components/rtm
        ./components/ww3
 
+You should now have a complete copy of the CESM2 source code in your /path/to/my_cesm_sandbox. 
+
 If there were problems obtaining an external, you might instead see something like:
 
 .. code-block:: console
 
    e-  ./components/cam
 
-For more information, run:
+This might happen if there was an unexpected interruption while downloading. In 
+which case, first try rerunning ``./manage_externals/checkout_externals``.
+If there is still a problem, try running with logging turned on using:
 
 .. code-block:: console
 
-   ./manage_externals/checkout_externals --help
-
-You should now have a complete copy of the CESM2 source code in your /path/to/my_cesm_sandbox. 
+   ./manage_externals/checkout_externals --logging
 
+Check the ``manage_externals.log`` file to see what errors are reported.
 
 Downloading input data
 ----------------------
 
-Input datasets are needed to run the model. CESM input data will be made
-available through a separate Subversion input data repository. The
-username and password for the input data repository will be the same as
-for the code repository for CESM registered users.
+Input datasets are needed to run the model. CESM input data are
+available through a separate Subversion input data repository.
 
 .. warning:: The input data repository contains datasets for many configurations and resolutions and is well over 10 TByte in total size. DO NOT try to download the entire dataset.
 
 Datasets can be downloaded on a case by case basis as needed and CESM
 provides tools to check and download input data automatically.
 
 A local input data directory should exist on the local disk, and it also 
-needs to be set in the CESM scripts via the variable ``$DIN_LOC_ROOT.``
+needs to be set in the CESM scripts via the variable ``DIN_LOC_ROOT.``
```

Comment:
I notice that ``$`` is removed here and from other ``DIN_LOC_ROOT`` instances. Did you want to put that back?

Reply:
fixed.